infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/LogReplicationClientConfigListener.java

```diff
@@ -129,16 +132,16 @@ private void handleRegistrationTableEntries(List<CorfuStreamEntry> registrationT
             log.warn("No client registration table entries found!");
             return;
         }
-
         for (CorfuStreamEntry entry : registrationTableEntries) {
             if (entry.getOperation().equals(CorfuStreamEntry.OperationType.UPDATE)) {
                 String clientName = ((ClientRegistrationId) entry.getKey()).getClientName();
                 ReplicationModel model = ((ClientRegistrationInfo) entry.getPayload()).getModel();
-                // TODO (V2): Currently we add a default subscriber for logical group use case instead of listening
-                //  on client registration. Subscriber should be added upon registration after grpc stream for session
-                //  creation is added.
-                // configManager.onNewClientRegister(subscriber);
-                // sessionManager.createOutgoingSessionsBySubscriber(subscriber);
+                LogReplication.ReplicationSubscriber subscriber = LogReplication.ReplicationSubscriber.newBuilder()
+                        .setClientName(clientName).setModel(model).build();
+                if (model.equals(ReplicationModel.LOGICAL_GROUPS)) {
+                    configManager.onNewClientRegister(subscriber);
+                    sessionManager.sendOutSessionToSinkSide(subscriber);
```

Comment:
the client registration itself doesn't necessarily lead to the start of a new session, it just means a new client is registered.

From LR Source's perspective, the start of a new session is: related client registered + supported topology received. It's some logic needs to be handled in `SessionManager` layer